### PR TITLE
Fix bugs and change texts

### PR DESCRIPTION
**Pull request by lukachi**

### CODE REVIEW

File: packages/snap/package.json [hunk 1]
```diff
@@ -44,6 +44,7 @@
     "@metamask/transaction-controller": "^27.0.1",
     "@qtumproject/qtum-wallet-connector": "workspace:^",
     "buffer": "6.0.3",
+    "eth-block-tracker": "^8.1.0",
```
Comment (KyrylR) [inline packages/snap/package.json:47]: ?

**Comment by KyrylR** [comment]
CI fails

**Comment by KyrylR** [comment]
Add Mainnet network address too

**Review by Baiqiang — APPROVED** [review]
LGTM!

**Review by KyrylR — APPROVED** [review]
Ok

**Pull request merged**